### PR TITLE
Customers table CRUD

### PR DESCRIPTION
**Pull request by shoshannaTM**

### CODE REVIEW

File: src/main/java/thomasmccue/dbclientapp/dao/CountryDao.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            ObservableList<String> results = FXCollections.observableArrayList();
+
+            try (Connection connection = JDBC.getConnection();
+                 PreparedStatement preparedStatement = connection.prepareStatement(sql)) {
+
+                try (ResultSet resultSet = preparedStatement.executeQuery()) {
+                    while (resultSet.next()) {
+                        results.add(resultSet.getString(1));
+                    }
+                }
```
Comment (gianlazz) [inline src/main/java/thomasmccue/dbclientapp/dao/CountryDao.java]: Curious about this additional try inside of the outer try and what the pattern is here? I see you seem to be using the initial try for the connection scope, do the ResultSet need to be closed too?

File: src/main/java/thomasmccue/dbclientapp/dao/FirstLevelDivisionDao.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        try (Connection connection = JDBC.getConnection();
+             PreparedStatement preparedStatement = connection.prepareStatement(sql)) {
+
+                preparedStatement.setInt(1, 1);
+
+                try (ResultSet resultSet = preparedStatement.executeQuery()) {
+                    while (resultSet.next()) {
+                        usList.add(resultSet.getString(1));
+                    }
+                }
```
Comment (gianlazz) [inline src/main/java/thomasmccue/dbclientapp/dao/FirstLevelDivisionDao.java]: Same here with the nested try blocks, are they necessary?

File: src/main/java/thomasmccue/dbclientapp/dao/FirstLevelDivisionDao.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                "FROM client_schedule.first_level_divisions WHERE Country_ID = ?;";
+        ObservableList<String> canList = FXCollections.observableArrayList();
+
+        try (Connection connection = JDBC.getConnection();
+             PreparedStatement preparedStatement = connection.prepareStatement(sql)) {
+
+            preparedStatement.setInt(1, 3);
+
+            try (ResultSet resultSet = preparedStatement.executeQuery()) {
+                while (resultSet.next()) {
```
Comment (gianlazz) [inline src/main/java/thomasmccue/dbclientapp/dao/FirstLevelDivisionDao.java]: Lots of duplication here, you could have another method that takes the country id and then you could centralize the logic and not need to be repeating yourself here for each country.

Comment (shoshannaTM) [reply src/main/java/thomasmccue/dbclientapp/dao/FirstLevelDivisionDao.java]: Fixed in my most recent commit to the branch

**Review by gianlazz — APPROVED** [review]

**Pull request merged**